code/modules/mob/dead/observer/spells.dm

```diff
@@ -26,6 +26,16 @@ var/global/list/boo_phrases=list(
 	range = 1 // Or maybe 3?
 
 /obj/effect/proc_holder/spell/aoe_turf/boo/cast(list/targets)
+
+	// First check if there are shadowlings nearby, because flickering lights will damage them.
+	var/shadowling_present = 0
+	for(var/mob/living/carbon/C in range(7, usr)) // (Range is Ghost Boo range (1) + Shadowling Veil range (5) + 1 for good measure)
+		if (C.get_species() == "Shadowling")
```

Comment:
I recommend checking for Shadowling, Lesser Shadowling (yes, that's a thing), and Shadow people. 

so `Shadowling`, `Lesser Shadowling`, and `Shadow`.